fail2ban/server/asyncserver.py

```diff
@@ -198,6 +201,7 @@ def close(self):
 	def stop(self):
 		self.close()
 
+	# better remains a method (not a property) since used as a callable for wait_for
```

Comment:
although we could as well make it an `active` property and then `lambda : x.active` it where waiting is needed.  Eventually we would need to make other parts the interface more pythonic